python/ray/tests/test_usage_stats.py

```diff
@@ -1165,6 +1165,13 @@ def ready(self):
         assert payload["python_version"] == python_version
         assert payload["schema_version"] == "0.1"
         assert payload["os"] == sys.platform
+        if sys.platform == "linux":
```

Comment:
I am confused about the test. Isn't this supposed to be `sys.platform != "linux"`?

Reply:
Good catch!

Reply:
Good quesiton